core/src/com/unciv/ui/utils/ExtensionFunctions.kt

```diff
@@ -93,6 +93,21 @@ fun Actor.addBorder(size:Float, color: Color, expandCell:Boolean = false): Table
     return table
 }
 
+/** Wrap an [Actor] in a [Group] of a given size */
+fun Actor.sizeWrapped(x: Float, y: Float): Group {
+    val wrapper = Group().apply {
+        isTransform = false // performance helper - nothing here is rotated or scaled
+        setSize(x, y)
+    }
+    this.apply {
```

Comment:
...why would you need this.apply?

Reply:
Oh. Lmao. …I stole/moved this from existing code, so in my fear of breaking anything, I tried to preserve the existing call stack apparently without paying any attention at all to what it was actually doing. Oops.